docs/overview/comparisons.md

```diff
@@ -0,0 +1,47 @@
+# Loki compared to other log systems
+
+## Loki / Promtail / Grafana vs EFK
+
+The EFK (Elasticsearch, Fluentd, Kibana) stack is used to ingest, visualize, and
+query for logs from various sources.
+
+Data in Elasticsearch is stored on-disk as unstructured JSON objects. Both the
+keys for each object and the contents of each key are indexed for searching.
+Data can then be queried using a JSON object to define a query (called the
+Query DSL) or through the Lucene query language if using Kibana.
```

Comment:
Es support SQL and lucene without kibana, kibana more like a web tool and UI for es i think.